config/tsconfig.workspace.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
```

Comment:
can maybe leave out `"composite": true,` since we got it from our tsconfig.base?